lib/crypto/packaged.py

```diff
@@ -115,7 +118,7 @@ def _no_sign(src, dest_path):
     # If this is a local development instance, just copy the file around
     # so that everything seems to work locally.
     log.info('Not signing the app, no signing server is active.')
-    with storage.open(dest_path, 'w') as dest_f:
+    with private_storage.open(dest_path, 'w') as dest_f:
```

Comment:
It's in `_no_sign()` so it does not matter for prod, but we probably want `public_storage` here. We are making a fake sign and the resulting package will be used as `File.signed_file_path`.